### PR TITLE
Added share button on page component

### PR DESCRIPTION
**Pull request by A-BMT02**
Main Changes :
1. Added Share button on Page Component .
2. Custom CSS styling for the button at index.css

### CODE REVIEW

File: src/pages/page.js [hunk 1]
```diff
@@ -2,13 +2,22 @@ import { useEffect, useState } from "react";
...
 export default function UserPage() {
   const { userId } = useParams();
   const { getUserDoc } = useFirestore();
   const [loading, setLoading] = useState(true);
 
   const [data, setData] = useState(null);
+  const [ buttonAnimate , setButtonAnimate ] = useState(false)
+  
+  const { state } = useAdmin();
+  const { username } = state;
```
Comment (nathgoutam93) [inline src/pages/page.js:20]: useAdmin hook will return the information of who is logged in , but in this case we do not need the username of the logged in person instead the username of the person we are currently visiting the page of. and that can be done by looking the userId parameter of the url . which is already done by userParams ,

File: src/pages/page.js [hunk 2]
```diff
@@ -21,16 +30,40 @@ export default function UserPage() {
 
   if (!data) return <h1>user not found</h1>;
 
+  const buttonAnimation = (e) => {
+    navigator.clipboard.writeText(`https://linkpile-bffd7.web.app/${username}`);
```
Comment (nathgoutam93) [inline src/pages/page.js:34]: change this username with userId that we are getting from the useParams()

File: src/pages/page.js [hunk 3]
```diff
@@ -21,16 +30,40 @@ export default function UserPage() {
...
+    console.log(`https://linkpile-bffd7.web.app/${username}`) ;
```
Comment (nathgoutam93) [inline src/pages/page.js:35]: remove this console log

File: src/pages/page.js [hunk 4]
```diff
@@ -2,13 +2,22 @@ import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import Page from "../components/page";
 import { useFirestore } from "../context/firestoreContext";
+import { IoShareOutline } from "react-icons/io5" ; 
+import { ShareAdmin } from "../components/share";
```
Comment (nathgoutam93) [inline src/pages/page.js:6]: share admin and BsAlignButton not in use remove those from import

**Review by nathgoutam93 — CHANGES_REQUESTED** [review]

**Comment by A-BMT02** [comment]
I merged our repositories and pulled. 
Seems like you took out all the code related to the share button

**Pull request merged**